sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -818,7 +819,7 @@ def _try_fuse_stages(a, b):
         for output_kv_coder_id in output_kv_coder_ids
     ]
     pack_output_value_coder = beam_runner_api_pb2.Coder(
-        spec=beam_runner_api_pb2.FunctionSpec(urn=common_urns.coders.KV.urn),
+        spec=beam_runner_api_pb2.FunctionSpec(urn=python_urns.tuple.KV.urn),
```

Comment:
Shouldn't this be `python_urns.TUPLE_CODER`?

Reply:
Thanks for catching that. Fixed.